lighthouse-core/report/html/renderer/category-renderer.js

```diff
@@ -356,6 +352,29 @@ class CategoryRenderer {
     return tmpl;
   }
 
+  /**
+   * Define the score arc of the gauge
+   * Credit to xgad for the original technique: https://codepen.io/xgad/post/svg-radial-progress-meters
+   * @param {SVGCircleElement} elem
+   * @param {number} percent
+   */
+  _setGaugeArc(elem, percent) {
+    const circumferencePx = 2 * Math.PI * Number(elem.getAttribute('r'));
+    // The rounded linecap of the stroke extends the arc past its start & end.
+    // First, we tweak the -90deg rotation to adjust
+    const strokeWidthPx = Number(elem.getAttribute('stroke-width'));
+    const rotationalAdjustmentPercent = 0.25 * strokeWidthPx / circumferencePx;
```

Comment:
maybe pull the `0.25` (and the `/2` below) into a constant for easy tweaking in the future?